dapp/src/utils/validation/shapes/wizard.js

```diff
@@ -25,8 +25,8 @@ export default object().noUnknown(false).shape({
   }),
   totalSupply: number().when(['existingToken', 'customToken'], {
     is: (existingToken, customToken) => (existingToken && existingToken.label && existingToken.value) || customToken,
-    then: number(),
```

Comment:
for adding a custom token totalSupply 0 is allowed.
Need to constrain just when token is created in the wizard

Reply:
Good point, thanks.